src-ghc-flat/Internal.hs

```diff
@@ -142,8 +142,8 @@ mkSubst' = foldr substSubst [] . mapMaybe mkSubst
 
 -- | Create simple substitution from type equalities
 mkSubst :: Ct -> Maybe ((TcTyVar, TcType),Ct)
-mkSubst ct@(CTyEqCan {..})  = Just ((cc_tyvar,cc_rhs),ct)
-mkSubst ct@(CFunEqCan {..}) = Just ((cc_fsk,TyConApp cc_fun cc_tyargs),ct)
+mkSubst ct@CTyEqCan{..}  = Just ((cc_tyvar,cc_rhs),ct)
+mkSubst ct@CFunEqCan{..} = Just ((cc_fsk,TyConApp cc_fun cc_tyargs),ct)
 mkSubst _                   = Nothing
```

Comment:
This change would need this line to be realigned too.

Reply:
A little oversight on my part.